setup.py

```diff
@@ -130,12 +131,12 @@ def get_extras_require() -> Dict[str, List[str]]:
             "pandas",
             "plotly>=4.0.0",
             "pytest",
-            "scikit-learn>=0.19.0,<0.23.0",
+            "scikit-learn>=0.19.0,<0.23.0 ; python_version<'3.9'",
```

Comment:
For python 3.9,  the constraint, `sklearn>0.23.0`, necessary because tests contain skopt's tests.

Reply:
Ah, I got it. Thank you for your investigation! I suppose so.